actionpack/lib/action_dispatch/testing/integration.rb

```diff
@@ -12,12 +12,12 @@ module RequestHelpers
       #
       # - +path+: The URI (as a String) on which you want to perform a GET
       #   request.
-      # - +parameters+: The HTTP parameters that you want to pass. This may
+      # - parameters: The HTTP parameters that you want to pass. This may
       #   be +nil+,
       #   a Hash, or a String that is appropriately encoded
       #   (<tt>application/x-www-form-urlencoded</tt> or
       #   <tt>multipart/form-data</tt>).
-      # - +headers_or_env+: Additional headers to pass, as a Hash. The headers will be
+      # - headers_or_env: Additional headers to pass, as a Hash. The headers will be
```

Comment:
Let's just call this headers. Hate that "or" setup. The headers get turned into env, that's fine. Don't need to spell that out further.

Actually, even better, let's do both, and just merge them together. Then it'll be even clearer in the code what the intent is.